templates/zerver/help/restore-the-last-unsent-message.md

```diff
@@ -1,12 +1,12 @@
 # Restore the last unsent message
 
-![Restore draft](/static/images/help/restore-draft.png)
-
 If you accidentally delete your message by clicking the **x**
 (<i class="icon-vector-remove"></i>) icon in the upper right corner of
-your messaging box, you can restore your previous unsent message by
-clicking a blue link labeled **Restore draft**, which appears between
-the **A** (<i class="icon-vector-font"></i>) and **eye**
-(<i class="icon-vector-eye-open"></i>) icons.
+your messaging box or by clicking outside the **Compose Message** box, you can
+restore the draft of your previous unsent message.
 
-**[Go to "Table of contents"](/help/#using-zulip)**
+To restore:
+1. Click a blue link labeled **Restore draft**, which appears between
```

Comment:
I don't think a step is necessary, since it's literally one step. You can just write "To restore your message draft draft, click the blue link labeled..."